.styleci.yml

```diff
@@ -1,14 +1,14 @@
 php:
-  preset: laravel
-  version: 8
-  disabled:
-    - no_unused_imports
-  finder:
-    not-name:
-      - index.php
-      - server.php
+    preset: laravel
+    version: 8
+    disabled:
+        - no_unused_imports
+    finder:
+        not-name:
+            - index.php
+            - server.php
 js:
-  finder:
-    not-name:
-      - webpack.mix.js
+    finder:
+        not-name:
+            - webpack.mix.js
 css: true
```

Comment:
@m4n1ok we might want to revert back to 2 spaces YAML 😅

Reply:
@luislavena, I cancelled changes made to this file